plugin-dev/Source/Sentry/Private/Linux/LinuxSentrySubsystem.cpp

```diff
@@ -15,6 +17,32 @@ void FLinuxSentrySubsystem::ConfigureDatabasePath(sentry_options_t* Options)
 	sentry_options_set_database_path(Options, TCHAR_TO_UTF8(*GetDatabasePath()));
 }
 
+void FLinuxSentrySubsystem::ConfigureCertsPath(sentry_options_t* Options)
+{
+	// In order to use CURL transport for sentry-native we have to manually specify path to a valid CA certificates on Linux.
+	// Unreal Engine itself follows a similar approach (see `CertBundlePath` implementation in CurlHttp.cpp for extra details)
+	static const char* KnownCertPaths[] =
+	{
+		"/etc/pki/tls/certs/ca-bundle.crt",
+		"/etc/ssl/certs/ca-certificates.crt",
+		"/etc/ssl/ca-bundle.pem"
+	};
+
+	for (const char* BundlePath : KnownCertPaths)
+	{
+		FString FileName(BundlePath);
+
+		if (FPaths::FileExists(FileName))
+		{
+			UE_LOG(LogSentrySdk, Log, TEXT("Sentry transport will use the certificate found at %s for verification."), *FileName);
+			sentry_options_set_ca_certs(Options, BundlePath);
+			return;
+		}
+	}
+
+	UE_LOG(LogSentrySdk, Warning, TEXT("Could not find CA certificates in any known location. Sentry transport may not function properly."));

```

Comment:
We could be more clear about what this means? Like "
```suggestion
	UE_LOG(LogSentrySdk, Warning, TEXT("Could not find CA certificates in any known location. Sentry transport may not function properly for handled events"));

```
? Is this the case?